core/assets/bundles/bundle_de.properties

```diff
@@ -74,25 +74,27 @@ players = {0} Spieler online
 players.single = {0} Spieler online
 server.closing = [accent]Schließe den Server ...
 server.kicked.kick = Du wurdest vom Server gekickt!
-server.kicked.whitelist = You are not whitelisted here.
+server.kicked.whitelist = Du bist hier nicht gewhitelistet.
 server.kicked.serverClose = Server geschlossen.
-server.kicked.vote = You have been vote-kicked. Goodbye.
+server.kicked.vote = Du wurdest, anhand einer Abstimmung gekickt!
+
+. Goodbye.
```

Comment:
This wouldn't display properly, since all the text has to be on one line (or separated by backslashes). For newlines, you can do the following:
`Du wurdest, anhand einer Abstimmung gekickt!\n\nGoodbye.`

Reply:
I see